locales/en/plugin__kubevirt-plugin.json

```diff
@@ -53,7 +53,7 @@
   "<0>Policy</0> {vmi?.status?.migrationState?.migrationPolicyName ? (\n                <ResourceLink\n                  groupVersionKind={MigrationPolicyModelGroupVersionKind}\n                  name={vmi.status.migrationState.migrationPolicyName}\n                />\n              ) : (\n                <MutedTextSpan text=\"No MigrationPolicy\" />\n              )}": "<0>Policy</0> {vmi?.status?.migrationState?.migrationPolicyName ? (\n                <ResourceLink\n                  groupVersionKind={MigrationPolicyModelGroupVersionKind}\n                  name={vmi.status.migrationState.migrationPolicyName}\n                />\n              ) : (\n                <MutedTextSpan text=\"No MigrationPolicy\" />\n              )}",
   "<0>Started</0> {vmi?.status?.migrationState?.startTimestamp &&\n                dateTimeFormatter.format(new Date(vmi?.status?.migrationState?.startTimestamp))}": "<0>Started</0> {vmi?.status?.migrationState?.startTimestamp &&\n                dateTimeFormatter.format(new Date(vmi?.status?.migrationState?.startTimestamp))}",
   "<0>Store the key in a project secret.</0><1>The key will be stored after the machine is created</1>": "<0>Store the key in a project secret.</0><1>The key will be stored after the machine is created</1>",
-  "<0>The descheduler can be used to evict a running pod to allow the pod to be rescheduled onto a more suitable node.</0><1></1><2>Note: if VirtualMachine has LiveMigratable=False condition, edit is disabled.</2>": "<0>The descheduler can be used to evict a running pod to allow the pod to be rescheduled onto a more suitable node.</0><1></1><2>Note: if VirtualMachine has LiveMigratable=False condition, edit is disabled.</2>",
+  "<0>The descheduler can be used to evict a running pod to allow the pod to be rescheduled onto a more suitable node.</0><1></1><2>Note: if VirtualMachine has <1>LiveMigratable=False </1> condition, edit is disabled.</2>": "<0>The descheduler can be used to evict a running pod to allow the pod to be rescheduled onto a more suitable node.</0><1></1><2>Note: if VirtualMachine has <1>LiveMigratable=False </1> condition, edit is disabled.</2>",
```

Comment:
Suggestion: "The descheduler evicts a running pod so that the pod can be rescheduled on a more suitable node. Note: This setting is disabled if the VirtualMachine is not live migratable."